src/sentry/utils/committers.py

```diff
@@ -322,7 +322,12 @@ def get_serialized_event_file_committers(
         return [
             {
                 "author": author,
-                "commits": [serialize(commit, serializer=CommitSerializer(exclude=["author"]))],
+                "commits": [
+                    serialize(
+                        commit,
+                        serializer=CommitSerializer(exclude=["author"], type="via SCM integration"),
```

Comment:
Are these types intended to behave like an enum or are they text to be presented to end users?

Reply:
I suppose either would work, but the end goal is to show it to users in the UI.

Reply:
Returning an enum value would make it simpler to add translations in the future. If the API endpoint is returning UI text, you're limited in what formatting can be done and we would need to add translations server side (if we ever get around to updating translations).